### PR TITLE
Fix table links

### LINKED ISSUE #31: Cursor isn't pointer inside of table cells that have links

### LINKED ISSUE #32: Clicking on a link in the table opens the page twice.

### PR DESCRIPTION
**Pull request by guywithnose**
Now the cells show the pointer cursor and they only open 1 window.

Fixes #32
Fixes #31

### CODE REVIEW

**Comment by nubs** [comment]
instead of doing it that way, why don't you just prevent default and/or stop propagation on the table cell clicks?

**Comment by guywithnose** [comment]
This seemed simpler to me because we only have to deal with 1 event.  I'll give it a try though and see what I can do.

**Comment by guywithnose** [comment]
Updated.  I do like this better because by leaving this links you can still see where you will be going in the browser corner.

File: js/main.js [hunk 1]
```diff
@@ -262,8 +262,8 @@
 
     dataTable.row.add([
       JSON.stringify(pullRequest),
-      '<a href="' + pullRequest.base.repo.html_url + '" target="_blank">' + pullRequest.base.repo.full_name + '</a>',
```
Comment (nubs) [inline js/main.js]: why no `target="_blank"`?

Comment (nubs) [inline js/main.js]: I guess because you were combining them... nm about that then :)

Comment (guywithnose) [inline js/main.js]: With the preventDefault I could leave it now, but it just doesn't do anything

Comment (guywithnose) [inline js/main.js]: I can't decide if it is more confusing with or without

**Comment by nubs** [comment]
I guess I still don't understand completely here.  Why do the `data-link` stuff?  why not just leave the href's and stuff so the links work as is.  Then your click handlers work exactly like they did before except that you ensure that it doesn't do the default href following and the javascript window clicking by only letting the one execute.

**Comment by guywithnose** [comment]
That is simpler

**Comment by nubs** [comment]
that looks right :) LGTM

**Pull request merged**